### PR TITLE
[FIX] sale_project: create project from SOL with allow_billable

### PR DESCRIPTION
**Pull request by damrOdoo**
This commit purpose is to ensure that projects created from a SOL
are created with `allow_billable` set to `True`

task-2992614

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by xavierbol — APPROVED** [review]
LGTM, thanks for the fix. 🙂

File: addons/sale_project/models/project.py [hunk 1]
```diff
@@ -333,6 +333,12 @@ def _get_profitability_sale_order_items_domain(self, domain=None):
             domain,
         ])
 
+    def _get_materials_action(self, record_ids):
+        args = ['other_revenues', [('id', 'in', record_ids)]]
+        if len(record_ids) == 1:
+            args.append(record_ids[0])
+        return {'name': 'action_profitability_items', 'type': 'object', 'args': json.dumps(args)}
+
```
Comment (IT-Ideas) [inline addons/sale_project/models/project.py:341]: Hello @damrOdoo 👋🏼 
I am so sorry, we forgot that you fixed that issue in this PR. So @xavierbol has made a fix we just merged today.
So your PR will only contain the `allow_billable`.
Can you update your commit message accordingly?

Thanks

**Review by IT-Ideas — COMMENTED** [review]
Thanks and all my apologies for the inconvenience!

File: addons/sale_project/models/sale_order_line.py [hunk 2]
```diff
@@ -150,6 +150,7 @@ def _timesheet_create_project_prepare_values(self):
             'sale_line_id': self.id,
             'active': True,
             'company_id': self.company_id.id,
+            'allow_billable': True,
```
Comment (IT-Ideas) [inline addons/sale_project/models/sale_order_line.py:153]: Could you rephrase the commit like this?
```
[FIX] sale_project: create project from SOL with allow_billable
This commit purpose is to ensure that projects created from a SOL
are created with `allow_billable` set to `True`

task-2992614
```
Thanks and sorry for being such a pain 🤣

**Review by IT-Ideas — COMMENTED** [review]
One last comment. Thanks for your work 🙂

**Comment by IT-Ideas** [comment]
Thanks 🙂 
@robodoo r+

**Comment by IT-Ideas** [comment]
@robodoo r+
